calcite/Calcite/Calcite.qml

```diff
@@ -26,12 +26,18 @@ import QtQuick
     {Calcite colors and themes}.
  */
 QtObject {
+    // Unknown was added it to more accurately match Application.styleHints.colorScheme
+    // This is an internal implementation detail.
+    // https://doc.qt.io/qt-6/qt.html#ColorScheme-enum
     enum Theme {
-        Light,
-        Dark
-    }
+    Unknown,
+    Light,
+    Dark
+}
+
+    property int theme: honorSystemTheme ? Application.styleHints.colorScheme : Calcite.Theme.Light
 
-    property int theme: Calcite.Theme.Light
+    property bool honorSystemTheme: true
```

Comment:
This is just wording preference, but I would go with `useSystemTheme` or something like that. I don't think `honorSystemTheme` is bad and I am good if we stick with it.

Reply:
> `honorSystemTheme` is bad

Say no more 😉 . In all seriousness, I like useSystemTheme better as well.